src/routes/jira/security/workspaces/containers/jira-security-workspaces-containers-post.test.ts

```diff
@@ -103,6 +137,10 @@ describe("Repositories Post", () => {
 	});
 
 	it("Should return an empty array if no matching repositories are found", async () => {
+		when(booleanFlag).calledWith(
+			BooleanFlags.ENABLE_GITHUB_SECURITY_IN_JIRA, jiraHost
+		).mockResolvedValue(true);
```

Comment:
This seems to be requirement for each unit test. I think, it would be nice move this piece of code to `beforeEach`